doc/content/gateways/lora-basics-station/cups.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Configuration and Update Server Protocol"
+description: ""
+weight: -1
+---
+
+This page contains information about connecting your gateway to {{% tts %}} using the Configuration and Update Server (CUPS) protocol.
```

Comment:
`connecting your gateway to {{% tts %}}` -> `managing your gateway using the  {{% tts %}} CUPS`